spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskTemplate.java

```diff
@@ -175,6 +175,13 @@ public void stop(String ids) {
 		restTemplate.postForLocation(executionLink.expand(ids).getHref(),values);
 	}
 
+	@Override
+	public void stop(String ids, String platform) {
+		MultiValueMap<String, Object> values = new LinkedMultiValueMap<>();
```

Comment:
MultiValueMap is for storing multiple values under the same key, that isn't needed here, a standard Map/HashMap will be sufficient.